manifests/t/TexasInstruments/TI-SmartView/TI-83/5.5.0.2092/TexasInstruments.TI-SmartView.TI-83.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TexasInstruments.TI-SmartView.TI-83
+PackageVersion: 5.5.0.2092
+PackageLocale: en-US
+Publisher: Texas Instruments Inc.
+PublisherUrl: https://education.ti.com
+PublisherSupportUrl: https://education.ti.com/en/customer-support
+PrivacyUrl: https://www.ti.com/legal/terms-conditions/privacy-policy.html
+Author: Texas Instruments Inc.
+PackageName: TI-SmartView CE Emulator Software for TI-83 Family

```

```suggestion
PackageName: TI-SmartView CE pour la famille TI-83
```
This must match Control Panel.

---

It seems that the TI-83 calculator serie is only available in France same as this software so I assume that's why the software name is in French. But it appears in English on the [download page](https://education.ti.com/fr/software/details/fr/61A623651E73418FBFFB545AB4B5A6AD/ti-smartview-ce-pour-la-famille-ti-83).